scylla-cdc-replicator/src/replicator_consumer.rs

```diff
@@ -9,7 +9,7 @@ use scylla::Session;
 use scylla_cdc::consumer::*;
 use std::sync::Arc;
 
-struct ReplicatorConsumer {
+pub(crate) struct ReplicatorConsumer {
```

Comment:
Fixes to this file should do not belong to this commit - this commit is supposed to only add replication_tests. Please move those changes into a separate commit and then squash the new commit with "Create a stub of the replicator."